core/embed/rust/src/ui/model_mercury/flow/util.rs

```diff
@@ -105,18 +106,13 @@ impl ConfirmBlobParams {
         self
     }
 
-    pub const fn with_info_button(mut self, info_button: bool) -> Self {
-        self.info_button = info_button;
-        self
-    }
-
     pub const fn with_verb(mut self, verb: Option<TString<'static>>) -> Self {
         self.verb = verb;
         self
     }
 
-    pub const fn with_verb_cancel(mut self, verb_cancel: Option<TString<'static>>) -> Self {
-        self.verb_cancel = verb_cancel;
+    pub fn with_verb_cancel(mut self, verb_cancel: Option<TString<'static>>) -> Self {
```

Comment:
could be `const`

Reply:
not anymore due to the `unwrap_or`.
you _could_ rewrite it as a match statement but it's not worth it, nobody is going to call this in a const context, right? (or if they do they can modify it)

Reply:
It has `.into()` so no...

Reply:
(`into` is a smaller problem because you can do `TString::from_translation` that is const. it's the `unwrap_or` that would need to be spelled out a different way)